components/policies/development/konflux-rbac/konflux-support-nslister-access/generate-support-nslister-clusterrolebinding-clusterpolicy.yaml

```diff
@@ -0,0 +1,52 @@
+---
+# This ClusterPolicy automatically generates a ClusterRoleBinding for users
+# in the 'konflux-support' Group to enable namespace-lister access.
+#
+# The namespace-lister service checks for ClusterRoleBindings with the label
+# 'namespace-lister.konflux-ci.dev/use-for-access: true' and grants access
+# to tenant namespaces based on individual User subjects in those bindings.
+
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: generate-konflux-support-nslister-clusterrolebinding
+  annotations:
+    policies.kyverno.io/title: "Generate ClusterRoleBinding for konflux-support Group Users"
+    policies.kyverno.io/category: Multi-Tenancy
+    policies.kyverno.io/description: >-
+      This policy automatically generates a ClusterRoleBinding for all users
+      in the 'konflux-support' Group. The ClusterRoleBinding includes the
+      label 'namespace-lister.konflux-ci.dev/use-for-access: true' which
+      enables namespace-lister to grant these users access to tenant namespaces.
+      The binding is synchronized with the Group, so any changes to group
+      membership are automatically reflected in the ClusterRoleBinding.
+spec:
+  rules:
+    - name: generate-nslister-clusterrolebinding
+      match:
+        any:
+        - resources:
+            kinds:
+            - user.openshift.io/v1/Group
+            names:
+            - konflux-support
+      context:
+        - name: userSubjects
+          variable:
+            jmesPath: "request.object.users[] | [].{kind: 'User', apiGroup: 'rbac.authorization.k8s.io', name: @}"
+      generate:
+        generateExisting: true
+        synchronize: true
+        apiVersion: rbac.authorization.k8s.io/v1
+        kind: ClusterRoleBinding
+        name: konflux-support-nslister
+        data:
+          metadata:
+            labels:
+              namespace-lister.konflux-ci.dev/use-for-access: 'true'
+          subjects: "{{ userSubjects }}"
+          roleRef:
+            kind: ClusterRole
+            name: view
```

Comment:
I'm not sure if we should use this one or the `everyone-can-view`. @hugares may I ask your feedback here?

Reply:
changed to konflux-viewer-user-actions, I believe this should be good enough, isn't it?